src/typed-method-types/workflows/triggers/webhook.ts

```diff
@@ -39,7 +39,7 @@ export type WebhookTriggerResponseObject<
     /**
      * @description The webhook object used to define the webhook trigger
      */
-    webhook?: string;
+    webhook?: { filter?: FilterType };
```

Comment:
From my tests the `triggers.create` call for a webhook trigger didn't include a `webhook` object and instead the `filter` is just returned at the top level as a sibling to `webhook_url`

```ts
     /**
     * @description The filter object used to define the webhook trigger
     */
    filter?: FilterType;
```

I'm having trouble testing `triggers.list` calls right now so can't validate that's the same but I'd expect it to be.